frontend/src/app/features/work-packages/components/wp-card-view/wp-single-card/wp-single-card.component.html

```diff
@@ -41,21 +44,21 @@
     <span [textContent]="wpProjectName(workPackage)"
           class="wp-card--project-name">
     </span>
-    <span [textContent]="wpTypeAttribute(workPackage)" 
+    <span [textContent]="wpTypeAttribute(workPackage)"
           class="wp-card--type"
           [ngClass]="typeHighlightingClass(workPackage)">
     </span>
     <a class="wp-card--id"
        [href]="fullWorkPackageLink(workPackage)"
        [ngClass]="uiStateLinkClass"
-       (accessibleClick)="emitStateLinkClicked(workPackage)"
-       [accessibleClickSkipModifiers]="true">
+       (click)="emitStateLinkClicked($event, workPackage)"
```

Comment:
Same here, not sure why the parent would need to know the page is going to change if they're going to be unloaded anyway. If they need to do some actions, they should probably be listening for router events instead. But I'm unsure what exactly this does, so feel free to leave it alone for now.

Reply:
That's because on bim, a click on the card will trigger different things compared to the regular page (e.g., open an associated viewpoint). That's why the click handling was basically moved up the chain to the place where this differentation is made (on the wp/bim page level).